src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,240 @@
+
+-  Duplicate the workflow you created
+-  Replace the workspace `id`
+
+## 4. Create a GitHub action
+
+The GitHub action described in this section updates the specified mesh when a PR is merged to the `main` branch of your repository. This step uses the variables and secrets you configured in the previous section.
+
+This workflow relies on two existing Adobe I/O GitHub Actions published on the GitHub Marketplace:
+
+-  [`adobe/aio-cli-setup-action`](https://github.com/adobe/aio-cli-setup-action) installs and configures the [CLI](https://github.com/adobe/aio-cli) on the GitHub infrastructure. For API Mesh, this action handles authentication.
+
+- [`adobe/aio-apps-action`](https://github.com/adobe/aio-apps-action) centralizes support for a GitHub workflow to leverage application-specific commands, such as deploying via `aio api-mesh:update`.
+
+In the `.github/workflows` folder, create a `.yml` file with the following contents:
+
+```yml
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  # push:
+  #   branches: [ "main" ]
+  # pull_request:
+  #   branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+   deploy:
+    name: Deploy to Prod
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        node-version: ['20']
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install
+        run: npm i
+      - name: Setup CLI
+        uses: adobe/aio-cli-setup-action@1.3.0
+        with:
+          os: ${{ matrix.os }}
+          version: 10.x.x
+      - name: api-mesh-plugin install
+        run: aio plugins:install @adobe/aio-cli-plugin-api-mesh
+      - name: Auth
+        uses: adobe/aio-apps-action@3.3.0
+        with:
+          os: ${{ matrix.os }}
+          command: oauth_sts
+          CLIENTID: ${{ secrets.CLIENTID_PROD }}
+          CLIENTSECRET: ${{ secrets.CLIENTSECRET_PROD }}
+          TECHNICALACCOUNTID: ${{ secrets.TECHNICALACCID_PROD }}
+          TECHNICALACCOUNTEMAIL: ${{ secrets.TECHNICALACCEMAIL_PROD }}
+          IMSORGID: ${{ secrets.IMSORGID_PROD }}
+          SCOPES: AdobeID, openid, read_organizations, additional_info.projectedProductContext, additional_info.roles, adobeio_api, read_client_secret, manage_client_secrets
+      - name: Select org
+        run: aio console:org:select ${{ secrets.ORGID_PROD }}
```

Comment:
We may need to help differentiate ORGID_PROD and IMSORGID_PROD with examples, as both are not same values. ORGID_PROD is the ID and IMSORGID_PROD is the code.

Reply:
@prabhuram93 two questions here looking at the download file from dev console: 

1. By `code` do you mean the `"code": "Adobe...",` in the `services` object?

2. Which ID in the following example is the appropriate org id for ORGID_PROD, `id` or `ims_org_id`?

```
  "org": {
      "id": "123456",
      "name": "adobe",
      "ims_org_id": "98765432123456789@AdobeOrg"
  },
```

Reply:
The ORGID_PROD should be "id": "123456"